charts/argo-cd/Chart.yaml

```diff
@@ -2,9 +2,12 @@ apiVersion: v2
 appVersion: v2.4.12
 description: A Helm chart for Argo CD, a declarative, GitOps continuous delivery tool for Kubernetes.
 name: argo-cd
-version: 5.4.4
+version: 5.4.5
 home: https://github.com/argoproj/argo-helm
 icon: https://argo-cd.readthedocs.io/en/stable/assets/logo.png
+sources:
+  - https://github.com/argoproj/argo-helm/tree/main/charts/argo-cd
```

Comment:
Can we just use argo-helm without reference to main branch?

Reply:
How? If you want to put a link to a special file or directory in Github, the branch name is always there?

Reply:
Now I've noticed this is link to argo-cd folder and not repository. Makes sense.